utils/newsTitles.js

```diff
@@ -8,40 +13,5 @@ export const newsTitles = [
     date: "19.02.2024",
     title: "Second Eclipse Tractus-X Community Days",
     blogLink: "/blog/community-days-05-2024"
-  },
```

Comment:
Are these entries deleted on purpose? If so, why? Personal opinion -> I think it's nice to see what has happened and been communicated.

Reply:
Forget my comment :) these are just the announcements on the homepage :) Thank you

Reply:
@maximilianong suggested to keep only latest 3

Reply:
There were pretty old news entries :D